src/constructApplications.js

```diff
@@ -6,7 +6,7 @@ import { resolvePath } from "./matchRoute";
  * loadApp: LoadApp;
  * }} ApplicationOptions
  *
- * @typedef {(applicationName: string) => Promise<import('single-spa').Application>} LoadApp
+ * @typedef {(config: import('single-spa').AppProps) => Promise<import('single-spa').Application>} LoadApp
```

Comment:
Just to make sure I understand this change. I think this is importing the type definition from single-spa itself and using it here?

Reply:
yes